CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+## Table of Contents
+
+- [v0.1.0](#v010)
+- [v0.2.0](#v020)
+
+# v0.2.0
+
+The v0.2.0 release is our first official minor release of the bpfd project. It includes:
+
```

Comment:
Do we want to mention that we've added support for TC and Tracepoint programs?  It's implied by the CRDs below but might be better to be explicit.

Reply:
Actually, should we have a section with the major new enhancements like:
- Support for TC and Tracepoint programs
- Kubernetes Operator and CRDs
- Improved documentation
- e2e testing and expanded integration tests
- Many bug fixes
- Others?

And then state what the release includes.

Reply:
Done! I just really included major new enhancements... Next release we'll actually be able to have a relevant change-log that's not the entire release history :)